### PR TITLE
Add OpenMW stable and nightly

### PR DESCRIPTION
**Pull request by apommel**
This PR adds the manifest for `openmw` and `openmw-nightly` packages ( #94 ). Tell me if anything seems wrong.

### CODE REVIEW

**Comment by Calinou** [comment]
Thanks for your contribution :slightly_smiling_face: 

I noticed nothing happens when starting OpenMW for the first time (no error dialog appears). Instead, I need to use the launcher or wizard to see some kind of window appear (telling me to install data files, as expected). Should we remove the "direct" OpenMW option from the start menu? It could confuse users as they're expected to go through the launcher first.

We could likely also remove the OpenMW Wizard start menu entry as people will only need to use it once, and it's already referenced in the Launcher.

File: bucket/openmw.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (Calinou) [inline bucket/openmw.json]: This file should use 4 spaces for indentation (like all other manifests).

Comment (apommel) [reply bucket/openmw.json]: I think this is because I used `scoop create` to generate this manifest. I correcte dit now, thanks.

**Comment by apommel** [comment]
I am not sure about removing the "direct" entry as after the config files have been generated with the launcher, it is the main way of launching the game. Do you think that adding a `notes` specifying that the user should launch the launcher first would be enough?

For the remaining parts, I agree with you. Do you think that would also be helpful to rename the shortcuts of the nightly (for example `OpenMW` -> `OpenMW (nightly)` ) in case both versions are installed?

**Comment by apommel** [comment]
I realized that the `checkver` had some issues, even with the github one. It was fixed with the latest commit.

**Comment by Calinou** [comment]
> I am not sure about removing the "direct" entry as after the config files have been generated with the launcher, it is the main way of launching the game. Do you think that adding a notes specifying that the user should launch the launcher first would be enough?

Yes, a `notes` property should suffice. Something like:

`Please run the OpenMW Wizard in the start menu to configure the game data path. Otherwise, OpenMW won't start correctly.`

**Comment by apommel** [comment]
> We should also find a way to rename the shim executables with a `-nightly` suffix so they don't conflict with the stable version. This way, people can install both the stable and nightly versions at the same time.

I was wondering about that too, I did it the [python27 way](https://github.com/ScoopInstaller/Versions/blob/master/bucket/python27.json). Everything should have been fixed in the latest commit.

**Comment by Calinou** [comment]
Thanks!

**Pull request merged**